tests/test_mobility_matrix.py

```diff
@@ -35,8 +35,8 @@ def test_mobility_matrix(Solver, periodicity, hydrodynamicRadius, numberParticle
     assert M.dtype == precision
     sym = M - M.T
     assert np.allclose(
-        sym, 0.0, rtol=0, atol=1e-7
-    ), f"Mobility matrix is not symmetric within 1e-6, max diff: {np.max(np.abs(sym))}"
+        sym, 0.0, rtol=0, atol=5e-5
```

Comment:
I want to specifically point out that the changes in the DPStokes parameter selection made the mobility matrix less symmetric. The DPStokes tests now only pass with a tolerance of 5e-5, as compared to the previous 1e-7.

Reply:
Hmmmm that's worth figuring out. Can you isolate *which* changes to the parameter selection made it less symmetric? Somehow I suspect it's the z-grid but it would also make sense it is was the kernel parameters

Reply:
I think it's both the z-grid and $m$. In this branch, I switched from $m=6$ to $m=4$ as we don't have torques implemented yet and that's what the paper recommends. Switching from 4 to 6 improves the worst-case error in how symmetric the mobility matrix is by 1-2 orders of magnitude, and doubling the number of points in the z-grid improves by another 1-2 orders of magnitude.

Reply:
Testing in double & single creates no noticeable difference, so I think you're right that it's just a tolerance issue.